### ISSUE TITLE
Block CLI generation: Extend Helper to make block var types better selectable and readable

### ISSUE THREAD

**Issue by dev7ch** [luyadev/luya#1523]
### What steps will reproduce the problem?

1.) generate a block with the command ./vendor/bin/luya cms/block/create
2.) follow the instructions until you can select the field type

### What is the expected result?

sth like this:

![block_cli_2](https://user-images.githubusercontent.com/15183757/30903175-23cbb114-a36e-11e7-88cb-4f49f9734639.gif)

### What do you get instead? (A Screenshot can help us a lot!)

- block var types not selectable via number.
- block var types in on singel line ( bad readability )

### Additional infos

extracted from this closed issue #1516 

| Q                | A
| ---------------- | ---
| LUYA Version     |  dev master

**Comment by nadar** [comment]
@dev7ch This makes it fast readable, but instead you have no idea what element this could be as the informations like: `slug`, `image-list`, `image` etc. are not available, you only have those explain texts which is absolute not clear what element this could be behind number 18, but `cms-page` is more clear.

**Comment by TheMaaarc** [comment]
I guess the solution is `[18] cms-page – Lorem ipsum`

**Comment by dev7ch** [comment]
@nadar yes as far i see we have $key => $value array like `slug => slug explanation` 

In my example the $key is replaced with a numeric value.

i agree it would better to include the var field name. Maybe just before the description in our $value field?
